packages/measure-tools/src/tools/MeasureAreaTool.ts

```diff
@@ -102,7 +102,7 @@ export class MeasureAreaTool extends MeasurementToolBase<AreaMeasurement, Measur
           hints.setRotation(mat.inverse()!);
       }
     }
-    hints.sendHints(true);
+    hints.sendHints(false);
```

Comment:
what are the points of hints if we dont show them? 
@a-gagnon @bsy-nicholasw

Reply:
spoke with nicholas about this, users need to manually turn on accusnap to see these now